services/api/mw2pdf/index.js

```diff
@@ -30,10 +32,20 @@ program
     // Process passthrough parameters
     let processedUrls = urls
     if (options.passthroughParameters) {
-      const passthroughParameters = JSON.parse(options.passthroughParameters)
-      processedUrls = processedUrls.map(
-        (url) => updateUrlParameters(url, passthroughParameters),
-      )
+      try {
+        assertValidPassthroughParameters(options.passthroughParameters)
```

Comment:
Would you be willing to add a comment here that says that this is the line that will cause the exception that we are handling? Just might make it easier for me when I come along later and try to understand the code after several weeks away.

Reply:
My only hesitation is that might become a stale comment -- in theory there are other things that might cause exceptions.

The use of the term `assert` is actually meant to convey that it can throw errors (the idea being that `assert` in node is a thing that will throw an error if it does not produce a truthy value.) but I understand if that isn't actually intuitive.

I'm going to look into tweaking this to make it clearer what's going on.

Reply:
I fixed this up by making two separate try / catch blocks.

The second one is just to be defensive.